flake.nix

```diff
@@ -32,15 +32,47 @@
           z3
           cvc5
           git
+        ] ++ lib.optionals stdenv.isDarwin [
           foundry.defaultPackage.${system}
         ];
 
         secp256k1-static = stripDylib (pkgs.secp256k1.overrideAttrs (attrs: {
           configureFlags = attrs.configureFlags ++ [ "--enable-static" ];
         }));
 
+        patchedHaskellPackages = pkgs.haskell.packages.ghc94.override {
+          overrides = self: super: {
+            # disable tests in optics
+            optics-core = pkgs.haskell.lib.dontCheck (self.callCabal2nix "optics-core" (pkgs.fetchFromGitHub {
```

Comment:
This will be included in the next bump right?

Reply:
Yes, when the next-next `haskell-updates` is merged after https://github.com/NixOS/nixpkgs/pull/236431